src/core.ts

```diff
@@ -51,29 +41,34 @@ export const syncIssue = async ({
       }
     }
   `,
-    { organization, number: project.number },
+    { organization, number },
   )
+}
 
-  /*
-
-    Step: Find the target field which we were given as input
+const fetchTargetField: (params: {
+  projectData: ProjectData
+  targetField: string
+  targetValue: string
+}) => Promise<{ targetFieldId: string; targetValueId: string }> = async ({
+  projectData,
+  targetField,
+  targetValue,
+}) => {
+  const targetFieldNode =
+    projectData.organization.projectNext.fields.nodes.find(
+      ({ name }) => name === targetField,
+    )
 
-  */
-  const targetField = projectData.organization.projectNext.fields.nodes.find(
-    ({ name }) => {
-      return name === project.targetField
-    },
-  )
   assert(
-    targetField,
-    `No field named "${project.targetField}" was found in the project`,
+    targetFieldNode,
+    `No field named "${targetField}" was found in the project`,
```

Comment:
Nit: I think `project.targetField` is more readable than `targetField`, but IMO it's not a big deal for such a short function. Same for other places where `project.targetFieldNode` was replaced by `targetFieldNode`.

Reply:
If this is were a longer function the reader might lose track of what `targetField` is (target field of what?) because that's only told in the function name, whereas keeping `project.targetField` gives a clear meaning to this field no matter where you're reading. For a smaller function it's not a big deal, but this idea is something to keep in mind regardless.